networking/eth2/src/integration-test/java/tech/pegasys/teku/networking/eth2/BeaconBlocksByRootIntegrationTest.java

```diff
@@ -241,14 +261,6 @@ public void requestBlockBySlot_withDisparateVersionsEnabled_requestAltairBlocks(
     }
   }
 
```

Comment:
Why don't we have deneb test here?

Reply:
same here as in  BeaconBlocksByRangeIntegrationTest.java, checked in requestBlockByRoot_withDisparateVersionsEnabled_requestNextSpecBlocks, good